Banking/src/main/java/com/example/Banking/model/Account.java

```diff
@@ -0,0 +1,130 @@
+package com.example.Banking.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Table(name = "accountdetails2")
```

Comment:
The name accountdetails2 doesn't sound good to me. It can be account_details, whenever theres a word break add underscore for table names.

Reply:
Okay